demoapp/.vscode/launch.json

```diff
@@ -1,7 +1,4 @@
 {
```

Comment:
Can we exclude this file in `.gitignore`?

Reply:
Sure, excluded